.flowconfig

```diff
@@ -1,7 +1,7 @@
 [ignore]
 .*/src/api/.*
 .*/src/core/.*
-.*/dist/.*
+.*/(?<!ripple-keypairs)/dist/.*
```

Comment:
I tried briefly with an online regex debug, before moving on. If you can confirm it works I'll change it.